rs/nns/governance/src/network_economics_tests.rs

```diff
@@ -6,6 +6,9 @@ use pretty_assertions::assert_eq;
 fn test_inherit_from_recursively() {
     let base = NetworkEconomics::with_default_values();
 
+    // This is tangential to the main point of this test.
+    assert_eq!(base.validate(), Ok(()));
```

Comment:
Nit. Seems like it would be better to factor this check out into its own tiny test, e.g., `test_default_values_valid`